packages/core-js/utils/logger.js

```diff
@@ -37,6 +39,11 @@ module.exports = {
     this._log(`${this.prepend}${msg}${completeStr}${this.postpend}`);
   },
 
+  async title(msg, font = 'small') {
```

Comment:
The logger now prints the figlet title.

I did this because I didn't want to duplicate code in the CLI and the DEPLOYER's titles.